designer/client/src/PageCreate.tsx

```diff
@@ -111,7 +114,9 @@ export class PageCreate extends Component<Props, State> {
     const payload = {
       title: title?.trim(),
       path: isQuestionPage ? `/${slugify(path)}` : defaults.path,
-      controller: controller ? defaults.controller : undefined
+      controller: controller ? defaults.controller : undefined,
+      repeatName:
+        controller === ControllerType.Repeat ? repeatName?.trim() : undefined
```

Comment:
Mind using the normalised `defaults`?

Hardly an issue for new controllers, but we have this PR outstanding to fully shift away from strings:

* https://github.com/DEFRA/forms-runner/pull/333

```suggestion
      repeatName:
        defaults.controller === ControllerType.Repeat ? repeatName?.trim() : undefined
```